<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_RATP_Window</class>
 <widget class="QDialog" name="Main_RATP_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RATP</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="ongStation">
      <attribute name="title">
       <string>&amp;Stations</string>
      </attribute>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>501</width>
         <height>261</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
      <widget class="QPushButton" name="ToNextPage_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>360</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="ongLines">
      <attribute name="title">
       <string>&amp;Lines</string>
      </attribute>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>11</y>
         <width>502</width>
         <height>403</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QRadioButton" name="radioBus">
                <property name="text">
                 <string>&amp;Bus</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioMetro">
                <property name="text">
                 <string>&amp;MÃ©tro</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioRer">
                <property name="text">
                 <string>&amp;RER</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="labelTransport">
                  <property name="text">
                   <string>&amp;Transport</string>
                  </property>
                  <property name="buddy">
                   <cstring>comboTransport</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboTransport"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="labelStation">
                  <property name="text">
                   <string>&amp;Station</string>
                  </property>
                  <property name="buddy">
                   <cstring>comboStation</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboStation"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushUniReq">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>OK</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widgetTimeView" native="true">
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>300</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
